test/bridge/utilities/Utilities.sol

```diff
@@ -0,0 +1,90 @@
+// SPDX-License-Identifier: MIT
+pragma solidity >=0.8.0;
+
+import "forge-std/Test.sol";
+
+interface IAccessControl {
+    function getRoleMember(bytes32 role, uint256 index)
+        external
+        view
+        returns (address);
+}
+
+interface IProxy {
+    function upgradeTo(address) external;
+}
+
+// Common utilities for forge tests
+contract Utilities is Test {
+    bytes32 internal nextUser = keccak256(abi.encodePacked("user address"));
```

Comment:
In general, prefer to use abi.encode() vs encodePacked unless carefully specifically trying to lower size of data - https://swcregistry.io/docs/SWC-133.

In this case, can just do: 
`bytes32 internal nextUser = keccak256(("user address");`

Reply:
Fixed in [bd4c33c](https://github.com/synapsecns/synapse-contracts/pull/126/commits/bd4c33c6d30ae7aacb3d0c57027a4d6497ae3329)